x-pack/plugins/security_solution/public/common/components/last_updated/index.test.tsx

```diff
@@ -0,0 +1,47 @@
+/*
```

Comment:
The files under `components/last_updated` were just moved from under timeline. Not new code.